docs/mr_proc/configs.md

```diff
@@ -0,0 +1,39 @@
+## Global files
+
+---
+
+mr_proc consists of two global files for specifying local paths and recruitment manifest 
+
+---
+
+### Global configs: `global_configs.json`
+   - This is a dataset specific file and needs to be modified based on local configs and paths
+   - This file is used as an input to the `run scripts` to read, process and track available data
+   - Copy, rename, and populate [sample_global_configs.json](https://github.com/neurodatascience/mr_proc/blob/main/sample_global_configs.json) 
+   - This file contains:
+      - Path to mr_proc_dataset
+      - List of pipelines + versions
+      - Path to local `container_store` comprising containers used by several workflow scripts
+
+!!! Suggestion
+
+    Although not mandatory, the preferred location would be: `<DATASET_ROOT>/proc/global_configs.json`
```

Comment:
```suggestion
    Although not mandatory, for consistency the preferred location would be: `<DATASET_ROOT>/proc/global_configs.json`.
```

Reply:
I initially thought the scripts are hard-wired to look for this config file in that location. That would probably be too constrained. But consistency is a good reason as well!